typings/index.d.ts

```diff
@@ -958,15 +958,15 @@ declare module 'discord.js' {
     public toString(): string;
   }
 
-  export class Message extends Base {
-    constructor(client: Client, data: object, channel: TextChannel | DMChannel | NewsChannel);
+  export class Message<G extends boolean = boolean> extends Base {
```

Comment:
maybe could rename the type parameter to like `Guild` or smth, G feels like a weird name

Reply:
But wouldn't that clash with the `Guild` type?

Reply:
true, the point was to have a better name for it tho

Reply:
I made the name `SentInGuild` and changed some other names too... Hopefully that has improved the situation.